examples/DynaFlow/IllustrativeExamples/HVDC_CLA/reference/outputs/curves/curves.csv

```diff
@@ -1,34 +1,34 @@
 time;NETWORK__BUS____7-BUS___10-2_AC_iSide2;HVDC_hvdc_P1Pu;
-0.000000;1377.507418;11.955000;
```

Comment:
Is it the only nrt impacted ?
What about the HVDC_PhaseShifter ?

Reply:
HVDC_PhaseShifter was not impacted according to the nrt report, I will double check

Reply:
Ok thanks!

Reply:
I confirm ;)